src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml.cs

```diff
@@ -69,11 +75,37 @@ internal partial class Dashboard : UserControl, IDisposable
             // for us to set up the AutomationPeer
             this.Loaded += Dashboard_Loaded;
 
+            if (editorFormatMapService != null)
+            {
+                _textFormattingMap = editorFormatMapService.GetEditorFormatMap("text");
+                _textFormattingMap.FormatMappingChanged += UpdateBorderColors;
+                UpdateBorderColors(this, eventArgs: null);
+            }
+
             this.Focus();
             textView.Caret.IsHidden = false;
             ShouldReceiveKeyboardNavigation = false;
         }
 
+        private void UpdateBorderColors(object sender, FormatItemsEventArgs eventArgs)
+        {
+            var resolvableConflictBrush = GetEditorTagBorderBrush(RenameFixupTag.TagId);
+            ResolvableConflictBorder.Stroke = resolvableConflictBrush;
+            ResolvableConflictText.Foreground = resolvableConflictBrush;
+
+            var unresolvableConflictBrush = GetEditorTagBorderBrush(RenameConflictTag.TagId);
+            UnresolvableConflictBorder.Stroke = unresolvableConflictBrush;
```

Comment:
This is for the border and text colors in the Dashboard Legend thing (see the dashboard screenshots above). We own that UI and can't push any of it up, unless we pushed the entire dashboard up.